src/main/scala/org/broadinstitute/dsde/firecloud/model/ElasticSearch.scala

```diff
@@ -45,6 +46,12 @@ case class ESType(`type`: String, fields: Map[String,ESInnerField], copy_to: Str
 object ESType extends ESPropertyFields {
   def apply(`type`: String):ESType = ESType(`type`, Map("completion" -> completionField(`type`)))
 }
+
+case class ESInternalType(
+  `type`: String,
+  analyzer: String = "whitespace",
+  include_in_all: Boolean = false) extends ESPropertyFields
+
```

Comment:
boy, I really wanna refactor ESType, ESAggregatableType, ESInternalType, and ESInnerField into a single class. I'm okay leaving this as-is for this PR ... maybe refactor afterwards.